static/app/utils/analytics/growthAnalyticsEvents.tsx

```diff
@@ -90,7 +90,7 @@ export type GrowthEventParameters = {
   };
   'growth.onboarding_clicked_instrument_app': {source?: string};
   'growth.onboarding_clicked_setup_platform_later': PlatformParam & {
-    project_index: number;
+    project_id: string;
```

Comment:
@scefali is it ok If I update this here? I think an id is even better than an index, isn't it?

Reply:
@priscilawebdev yea agreed